src/docfx/restore/index/DependencySlot.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Newtonsoft.Json;
+
+namespace Microsoft.Docs.Build
+{
+    internal abstract class DependencySlot
+    {
+        public string Id { get; set; }
+
+        public string Url { get; set; }
+
+        public DateTime LastAccessDate { get; set; }
```

Comment:
why is there a need to store time?

Reply:
it's last access data, used for ordering and apply lockdown time